src/pages/OppdragPage.tsx

```diff
@@ -45,34 +47,51 @@ export default function OppdragPage() {
                 )}
               </div>
             </div>
-            <div className={commonstyles.buttonrowLeft}>
-              <Suspense
-                fallback={
-                  <Button size="small" loading variant="secondary-neutral">
-                    Omposteringer
-                  </Button>
-                }
-              >
-                <OmposteringModal oppdragsId={oppdrag!.oppdragsId} />
-              </Suspense>
-              <Suspense
-                fallback={
-                  <Button size="small" loading variant="secondary-neutral">
-                    Status historikk
-                  </Button>
-                }
-              >
-                <StatushistorikkModal oppdragsId={oppdrag!.oppdragsId} />
-              </Suspense>
-              <Suspense
-                fallback={
-                  <Button size="small" loading variant="secondary-neutral">
-                    Enhetshistorikk
-                  </Button>
-                }
-              >
-                <EnhetshistorikkModal oppdragsId={oppdrag!.oppdragsId} />
-              </Suspense>
+            <div className={commonstyles.leftandrightbuttons}>
+              <div className={commonstyles.buttonrowLeft}>
+                <Suspense
+                  fallback={
+                    <Button size="small" loading variant="secondary-neutral">
+                      Omposteringer
+                    </Button>
+                  }
+                >
+                  <OmposteringModal oppdragsId={oppdrag!.oppdragsId} />
+                </Suspense>
+                <Suspense
+                  fallback={
+                    <Button size="small" loading variant="secondary-neutral">
+                      Status historikk
+                    </Button>
+                  }
+                >
+                  <StatushistorikkModal oppdragsId={oppdrag!.oppdragsId} />
+                </Suspense>
+                <Suspense
+                  fallback={
+                    <Button size="small" loading variant="secondary-neutral">
+                      Enhetshistorikk
+                    </Button>
+                  }
+                >
+                  <EnhetshistorikkModal oppdragsId={oppdrag!.oppdragsId} />
+                </Suspense>
+              </div>
+              <div className={commonstyles.knapperad}>
+                <Button
+                  size={"small"}
+                  variant={"secondary-neutral"}
+                  onClick={() =>
+                    downloadAsCsv(
+                      gjelderId,
+                      oppdrag!.navnFagOmraade,
+                      oppdragsLinjer,
+                    )
+                  }
+                >
+                  Til Excel <FileCsvIcon title="a11y-title" fontSize="1rem" />
```

Comment:
```suggestion
                  Button icon={<FileCsvIcon title="Til Excel" fontSize="1rem" />}
```
Kanskje bruke icon prop'en?